src/wstool/config_yaml.py

```diff
@@ -355,7 +355,10 @@ def get_path_spec_from_yaml(yaml_dict):
             elif key == "uri":
                 uri = value
             elif key == "version":
-                version = value
+                if value is not None: 
+                    version = str(value)
+                else: 
+                    version = value
```

Comment:
is the else branch needed: Version is already None?

Reply:
Removed